### ISSUE TITLE
Color of "details" links in dialogs do not meet minimum contrast ratio threshold

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#1224]
### Problem

The color of the "Details" links in this screenshot:

![Image](https://github.com/user-attachments/assets/40f17ad8-855c-497b-ba29-dd955e0a5688)

do not meet minimum contrast ratio thresholds. The color is defined in https://github.com/opencast/opencast-admin-interface/blob/main/src/styles/components/modals/_modal-components.scss#L295 and use `l-blue` which is defined as:

```
$primary-color-light-blue:     #378dd4;
$l-blue:                       $primary-color-light-blue;
 ```

The contrast ratio is not enough:

![Image](https://github.com/user-attachments/assets/d95411ba-6cdc-4812-aa75-191b7a8ed47d)

### Proposed solution

Converting to Oklab color space and maintaining the overall chroma but decreasing the lightness to match the desired color contrast ratio the color of the "details" link should be changed from "#378dd4" to "#207bc0". The contrast ratio would then change to:

![Image](https://github.com/user-attachments/assets/71593018-ca99-4dfd-a474-badded31f075)

And the colors are still very similar:

![Image](https://github.com/user-attachments/assets/aaccf59d-5c38-4281-8325-1c23083c117e)
